toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/LocalSceneView.kt

```diff
@@ -62,7 +64,37 @@ import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 
 /**
- * A compose equivalent of the view-based [LocalSceneView].
+ * A user interface control that displays geographic content defined by a local [ArcGISScene].
+ * A local scene view is a user interface that displays layers and graphics. It controls the area of
+ * the local scene that is visible and supports user interactions such as pan and zoom. The local
+ * scene view also provides access to the underlying layer data in a local scene.
+ *
+ * To display a local scene, add this view to your app and assign a local scene to it via
+ * [LocalSceneView.scene]. This loads the scene and its content, such as a [Basemap] and a collection
+ * of operational layers, and displays this content on screen.
+ *
+ * User interactions such as pan, zoom, rotate, and identify or selection are supported in the
+ * [LocalSceneView] using touch, mouse, keyboard or pen/pencil. If required, you can override keys,
+ * clicks, and gestures to provide a specific user experience.
+ *
+ * The visible area ([Viewpoint]) of the local scene view is defined by the visible extent of the
+ * scene. To determine the current visible area or the center point and scale of the view, make sure
+ * that any user-initiated or programmatic navigation is complete before getting the current
+ * [Viewpoint] by calling [GeoView.isNavigating].
+ *
+ * You can programmatically set the visible area by specifying a viewpoint. For example,
+ * [GeoView.setViewpoint] sets the visible area to the extent of a provided geometry, and
+ * [LocalSceneView.setViewpointCamera] centers the local scene view at a given point. Any geometries
+ * passed to these methods are automatically projected to match the [SpatialReference] of the
+ * [LocalSceneView.scene], if required.
```

Comment:
This needs to be adapted to compose:
```suggestion
 * You can programmatically set the visible area by specifying a viewpoint. For example,
 * [GeoViewProxy.setViewpoint] sets the visible area to the extent of a provided geometry, and
 * [LocalSceneViewProxy.setViewpointCamera] centers the local scene view at a given point. Any geometries
 * passed to these methods are automatically projected to match the [SpatialReference] of the
 * [arcGISScene], if required.
```